source/renderer/zettlr-editor.js

```diff
@@ -447,30 +453,35 @@ class ZettlrEditor {
     this._cm.on('renderLine', (cm, line, elt) => {
       let monospaceWidth = this.computeMonospaceWidth()
       let spaceWidth = this.computeSpaceWidth()
+      let tabWidth = spaceWidth * 5
```

Comment:
Are you absolutely sure it's 5? Quite an unusual number.

Reply:
To be honest, I've no idea why it's 5. Was try and error